R-package/vignettes/discoverYourData.Rmd

```diff
@@ -51,24 +48,24 @@ A *categorical* variable has a fixed number of different values. For instance, i
 >
 > Type `?factor` in the console for more information.
 
-To answer the question above we will convert *categorical* variables to `numeric` one.
+To answer the question above we will convert *categorical* variables to `numeric` ones.
 
 ### Conversion from categorical to numeric variables
 
 #### Looking at the raw data
 
-In this Vignette we will see how to transform a *dense* `data.frame` (*dense* = few zeroes in the matrix) with *categorical* variables to a very *sparse* matrix (*sparse* = lots of zero in the matrix) of `numeric` features.
++In this Vignette we will see how to transform a *dense* `data.frame` (*dense* = the majority of the matrix is non-zero) with *categorical* variables to a very *sparse* matrix (*sparse* = lots of zero entries in the matrix) of `numeric` features.
 
 The method we are going to see is usually called [one-hot encoding](https://en.wikipedia.org/wiki/One-hot).
 
-The first step is to load `Arthritis` dataset in memory and wrap it with `data.table` package.
+The first step is to load the `Arthritis` dataset in memory and wrap it with the `data.table` package.
 
 ```{r, results='hide'}
 data(Arthritis)
 df <- data.table(Arthritis, keep.rownames = FALSE)
 ```
 
-> `data.table` is 100% compliant with **R** `data.frame` but its syntax is more consistent and its performance for large dataset is [best in class](https://stackoverflow.com/questions/21435339/data-table-vs-dplyr-can-one-do-something-well-the-other-cant-or-does-poorly) (`dplyr` from **R** and `Pandas` from **Python** [included](https://github.com/Rdatatable/data.table/wiki/Benchmarks-%3A-Grouping)). Some parts of **XGBoost** **R** package use `data.table`.
+> `data.table` is 100% compliant with **R** `data.frame` but its syntax is more consistent and its performance for large dataset is [best in class](http://stackoverflow.com/questions/21435339/data-table-vs-dplyr-can-one-do-something-well-the-other-cant-or-does-poorly) (`dplyr` from **R** and `Pandas` from **Python** [included](https://github.com/Rdatatable/data.table/wiki/Benchmarks-%3A-Grouping)). Some parts of **XGBoost's** **R** package use `data.table`.
```

Comment:
Why remove the TLS address?

Reply:
Sorry, I think that was a typo.  I reverted that change back to https